source/bsondump.txt

```diff
@@ -37,116 +41,21 @@ to format its data.
-
-   - :v4.2:`MongoDB 4.2 bsondump <reference/program/bsondump>`
-   - :v4.0:`MongoDB 4.0 bsondump <reference/program/bsondump>`
+Compatibility and Platform Support
+----------------------------------
 
-This documentation is for version ``{+release+}`` of |tool-binary|. 
-
-Compatibility
--------------
-
-MongoDB Server Compatibility
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. include:: /includes/extracts/dbtools-compatibility-single.rst
-
-Platform Support
-~~~~~~~~~~~~~~~~
-
-|tool-binary| version ``{+release+}`` is supported on the following
-platforms:
-
-.. include:: /includes/fact-platform-support.rst
+For |tool-binary| compatibility and platform support, see
+:ref:`bsondump-compatibility-and-platform-support`.
 
 Installation
 ------------
 
-.. include:: /includes/fact-see-install-guide.rst
+To install |tool-binary|, see :ref:`bsondump-installation`.
 
 Options
 -------
 
-.. option:: --help
-
-   Returns information on the options and use of :program:`bsondump`.
-
-
-.. option:: --verbose, -v
-
-   Increases the amount of internal reporting returned on standard output
-   or in log files. Increase the verbosity with the ``-v`` form by
-   including the option multiple times, (e.g. ``-vvvvv``.)
-
-
-.. option:: --quiet
-
-   Runs :program:`bsondump` in a quiet mode that attempts to limit the amount
-   of output.
-
-   This option suppresses:
-   
-   - output from :term:`database commands <database command>`
-   
-   - replication activity
-   
-   - connection accepted events
-   
-   - connection closed events
-
-
-.. option:: --version
-
-   Returns the :program:`bsondump` release number.
-
-
-.. option:: --objcheck
-
-   Validates each :term:`BSON` object before outputting it in :term:`JSON`
-   format. By default, :program:`bsondump` enables :option:`--objcheck <bsondump --objcheck>`.
-   For objects with a high degree of sub-document nesting,
-   :option:`--objcheck <bsondump --objcheck>` can have a small impact on performance.
-
-
-.. option:: --type=<json|debug>
-
-   Changes the operation of :program:`bsondump` from outputting
-   ":term:`JSON`" (the default) to a debugging format.
-
-
-.. option:: --pretty
-
-   Outputs documents in a pretty-printed format JSON.
-
-
-.. option:: --bsonFile
-   
-   Specifies the path to a :term:`BSON` file to dump to JSON. :option:`--bsonFile` is
-   an alternative to the positional :option:`\<bsonFilename\>` option.
-   
-   By default, :program:`bsondump` reads from standard input.
-
-
-.. option:: <bsonFilename>
-
-   The final argument to :program:`bsondump` is a document containing
-   :term:`BSON`. This data is typically generated by
-   :program:`bsondump` or by MongoDB in a :term:`rollback` operation.
-
-
-.. option:: --outFile=<file>
-   
-   Specifies the path of the file to which :program:`bsondump` should write
-   its output JSON data.
-   
-   By default, :program:`bsondump` writes to standard output.
-
+For |tool-binary| options, see :ref:`bsondump-options`.
 
 Examples
```

Comment:
I thought we were going to put the examples in a separate page as well, given the size of the examples for some of the larger pages. WDYT?